data/json/items/comestibles.json

```diff
@@ -11311,5 +11311,23 @@
         "fun": 10,
         "spoils_in": 24,
         "flags": [ "EATEN_HOT" ]
+    },
+    {
+        "id": "sausage_wasteland",
+        "copy-from": "offal",
+        "type": "COMESTIBLE",
+        "name": "wasteland sausage",
+        "description": "Lean sausage made from heavily salt-cured offal, with natural gut casing.  Waste not, want not.",
+        "charges": 6,
+        "stack_size": 6,
```

Comment:
Are you sure you need to specify stack_size identical to charges? As far as I recall, it defaults to charges. If it does, it shouldn't be explicitly set.